### PR TITLE
Fix mouse events occurring on children of a disabled form field

### PR DESCRIPTION
**Pull request by SteveSandersonMS**
Fixes https://github.com/aspnet/AspNetCore/issues/16655

Our event delegation system, like other JS event delegation systems, tries to replicate normal DOM event handling behavior, and this was one relatively obscure case that had been missed.

### CODE REVIEW

**Comment by SteveSandersonMS** [comment]
cc @Pilchie for approval

**Review by javiercn — APPROVED** [review]
Quick, tested and complete. Great fix!

File: src/Components/Web.JS/src/Rendering/EventDelegator.ts [hunk 1]
```diff
@@ -21,6 +21,8 @@ const nonBubblingEvents = toLookup([
   'DOMNodeRemovedFromDocument',
 ]);
 
+const disableableEventNames = toLookup(['click', 'dblclick', 'mousedown', 'mousemove', 'mouseup']);
```
[the remark this answers is not in the document]
Comment (SteveSandersonMS) [reply src/Components/Web.JS/src/Rendering/EventDelegator.ts]: Experimentation and what I can piece together from docs

**Comment by Pilchie** [comment]
Approved for 3.1.0-Preview3.

**Pull request merged**